index.js

```diff
@@ -22749,7 +22749,7 @@ function BeeSwarmSimulator(DATA){
 
                             let taskAction = MATH.doStatGrammar(out.quests[i].req[j][0])
                             let taskAmount = MATH.addCommas(out.quests[i].req[j][1].toString())
-                            let taskFocus = 'ANOTHER TEST'
+                            let taskFocus =  out.quests[i].req[j][1]>1&&MATH.doStatGrammar(out.quests[i].req[j][0])==='Defeat'?MATH.doPlural(MATH.doGrammar(out.quests[i].req[j][0])):MATH.doGrammar(out.quests[i].req[j][0])

```

Comment:
This should update as the task is being completed. If not, then assigning the value to a variable, somehow, would interfere with the ability of the value to update in real time when inserted into text.